simpleclient_spring_boot/src/test/java/io/prometheus/client/spring/boot/PrometheusEndpointTest.java

```diff
@@ -55,6 +53,35 @@ public void testMetricsExportedThroughPrometheusEndpoint() {
         matchesPattern("foo_bar\\{label1=\"val1\",label2=\"val2\",?\\} 3.0")));
   }
 
+  @Test
+  public void testMetricsExportedThroughPrometheusMVCEndpoint() {
+    // given:
+    final Counter promCounter = Counter.build()
+            .name("foo_bar2")
+            .help("a simple prometheus counter")
+            .labelNames("label1", "label2")
+            .register();
```

Comment:
I recommend not registering with the default registry otherwise tests will leak metrics across.  (Likely the reason for the @Ignore on the other tests)